### PR TITLE
Github Actions workflow implementation

### PR DESCRIPTION
**Pull request by pavlua**
What was done:
1) bumped target framework versions to actual ones
2) added github actions workflows
3) remove old azure pipelines yamls

For the publication of packages to nuget.org to work correctly, you need to add a secret named NUGET_APIKEY in the GitHub repository to store the API key (https://github.com/progaudi/MsgPack.Light/settings/secrets/actions).

There are 2 workflows:
- ci.yml: triggers when PR to master was created or updated. It builds and run autotests and as result it pushes a new prerelease package to nuget.org. Version calculates using git describe --tags (example 1.6.2.8-g6ed8, where 1.6.2 is last tag, and we got 8th commit since this tag). Example of successful run from work branch https://github.com/pavlua/MsgPack.Light/actions/runs/6024480052 
- bump-version-and-publish.yml: triggers when PR was merged to master. It pushes a new tag into repository. Tag is calculated as incremented semver using https://github.com/anothrNick/github-tag-action. By default, it increments patch version. We can increment another minor or major also (https://github.com/anothrNick/github-tag-action#bumping). After pushing the tag into repo, it builds, test and publish release version to nuget.org. Example of successful run from work branch https://github.com/pavlua/MsgPack.Light/actions/runs/6024496914

### CODE REVIEW

File: .github/workflows/bump-version-and-publish.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            3.1.x
+            6.0.x
+      - name: Restore dependencies
+        run: dotnet restore
+      - name: Build
+        run: dotnet build --no-restore -c Release
+      - name: Test on .netcoreapp3.1
+        run: dotnet test -c Release -f netcoreapp3.1 --no-build --verbosity normal
+      - name: Test on .net6.0
+        run: dotnet test -c Release -f netcoreapp3.1 --no-build --verbosity normal
```
Comment (aensidhe) [inline .github/workflows/bump-version-and-publish.yml]: Copy/paste error here and below. Probably, we need net6.0 here.

**Review by aensidhe — APPROVED** [review]
Otherwise looks good. Now I'll have more time on daily basis to look on PRs.

File: .github/workflows/bump-version-and-publish.yml (hunk 1 above)
Comment (pavlua) [reply .github/workflows/bump-version-and-publish.yml]: Yes, thank you for noticing, fixed.

Comment (pavlua) [reply .github/workflows/bump-version-and-publish.yml]: @aensidhe if everything looks good, I think this PR is ready to be merged

**Review by aensidhe — APPROVED** [review]

**Comment by aensidhe** [comment]
@pavlua I've returned windows build. Thanks for giving me ideas.

**Pull request merged**